src/ipyradiant/basic_tools/selection_widget.py

```diff
@@ -42,12 +67,33 @@ def _make_selected_things(self):
     @trt.default("add_button")
     def _make_add_button(self):
         return ipyw.Button(
-            description="Add",
+            description="Add -->".center(12),
         )
 
     @trt.default("remove_button")
     def _make_remove_button(self):
-        return ipyw.Button(description="Remove")
+        return ipyw.Button(description="<-- Remove".center(12))
+
+    @trt.observe("data")
+    def _change_data(self, change):
+        self.available_things_list = self._make_available_things_list()
+
+    @trt.observe("available_things_list")
+    def _change_selector(self, change):
+        self.available_things = self._make_available_things()
+
+    @trt.observe("selected_things_list")
+    def _change_selected_items(self, change):
+        self.selected_things = self._make_selected_things()
+
+    @trt.observe(
+        "selected_things", "available_things", "right_panel_text", "left_panel_text"
+    )
+    def _change_third_column(self, change):

```

Comment:
right panel, left panel, third column

The naming is a bit confusing. It is not obvious to me what this method is doing based on the name and the code. Maybe a docstring for the method and/or rename the function?